src/actions/search.js

```diff
@@ -32,19 +32,40 @@ const searchVotes = ({ address }) =>
       }));
   };
 
-const searchVoters = ({ address, publicKey }) =>
+const searchVoters = ({
+  address, publicKey, offset, limit, append,
+}) =>
   (dispatch, getState) => {
     const activePeer = getState().peers.data;
-    getVoters(activePeer, publicKey).then(response =>
+    getVoters(activePeer, {
+      publicKey, offset: (offset || 0), limit: (limit || 100),
```

Comment:
Why the default values here again? You have them already in the definition of `getVoters` in `src/utils/api/delegate.js`
```suggestion
      publicKey, offset, limit,
```